### DISCUSSION TITLE
请问如下拉取2.4版本 standalone的docker image

### DISCUSSION THREAD

**Discussion by sj741231** [milvus-io/milvus#34271] [Q&A and General discussion]
我们之前使用2.2.9，现在希望使用2.4以上版本，但是docker pull  milvusdb/milvus:v2.4.X，全都无法拉取镜像。我试了3个源都不行？请问从哪里可以下载2.4的docker镜像？

**Comment by JackLCL** [comment]
@sj741231 Please send an email to community@zilliz.com, you will get the latest milvus docker image.
![image](https://github.com/milvus-io/milvus/assets/53512883/ce2705a0-39e9-4f94-8c5a-2be054250816)
